cypress/integration/home_page.spec.js

```diff
@@ -3,9 +3,18 @@
 import 'cypress-audit/commands'
 
 describe('Home page', () => {
-  it('Displays all necessary components on the home page', () => {
-    cy.visit('/')
+  beforeEach(() => {
+    cy.getCookies().should('be.empty')
+    cy.setCookie('hackneyToken', Cypress.env('GSSO_TEST_KEY'))
+    cy.getCookie('hackneyToken').should(
+      'have.property',
+      'value',
+      Cypress.env('GSSO_TEST_KEY')
+    )
+    cy.visit(Cypress.env('HOST'))
+  })
```

Comment:
Hey as mentioned (for another PR anyway) we can put this into a helper function as all tests will probably need to use this